src/stream/frame/content.rs

```diff
@@ -424,6 +424,18 @@ pub fn decode(
             Ok(content)
         }
         "IPLS" | "IPL" | "TMCL" | "TIPL" => decoder.involved_people_list(),
+        // According to the ID3v2.2.0/ID3v2.3.0 specifications, these text frames may contain
+        // multiple values that 'are seperated with the "/" character'. Hence, the "/" character is
+        // replaced with a null byte for these frames, so that the values can be accessed
+        // separately using the `text_values()` method.
+        //
+        // All other frames do not support multiple values in these ID3 versions and therefore their
+        // contents are read verbatim. Note that when trying to write multiple values, the values
+        // will be joined using "/" for  *all* tags, because the alternative would be to just throw
+        // an error.
+        "TCOM" | "TCM" | "TEXT" | "TXT" | "TOLY" | "TOL" | "TOPE" | "TOA" | "TPE1" | "TP1" => {
```

Comment:
Note that only `TPE1` but not`TPE2`/`TPE3`/`TPE4` are included here. This is intentional, because the slash is only mentioned for `TPE1`:

> TPE1
> The 'Lead artist(s)/Lead performer(s)/Soloist(s)/Performing group' is used for the main artist(s). They are seperated with the "/" character.
>
> TPE2
> The 'Band/Orchestra/Accompaniment' frame is used for additional information about the performers in the recording.
>
> TPE3
> The 'Conductor' frame is used for the name of the conductor.
>
> TPE4
> The 'Interpreted, remixed, or otherwise modified by' frame contains more information about the people behind a remix and similar interpretations of another existing piece.
```